app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -4,8 +4,9 @@ import NarrowContainer from "~/components/layout/NarrowContainer";
 import { Divider } from "~/components/layout";
 import DiscoveryHeader from "../discovery/DiscoveryHeader";
 import ProjectsView from "../projects/ProjectsView";
+import VisualizationsView from "../visualizations/VisualizationsView";
```

Comment:
for imports you may want to stick with the ~ aliases in webpack.config.common.js or define some more

Reply:
instead of paths such as `"../visualizations/VisualizationsView"`?

Reply:
yes for ease of moving/not having relative paths, but minor/up for debate

Reply:
ok I'll leave it for now